### ISSUE TITLE
Not able to load debug scripts in sharepoint

### ISSUE THREAD

**Issue by maheshwaript13ec** [SharePoint/sp-dev-docs#5570]
## Category

- [x] Bug

## Describe the bug

> **DELETE THIS LINE BEFORE SUBMITTING** - Provide a clear & concise description of what the bug is. Please follow our guidance on [How To Create Good Issues](https://github.com/sharepoint/playground-repo/wiki/How-to-Create-Good-Issues) which explains how to apply formatting, adding references & resources, screenshots, etc.

## Steps to reproduce

> **DELETE THIS LINE BEFORE SUBMITTING** - Provide steps to reproduce the behavior:

1. Click on '....'
1. Scroll down to '....'
1. See error

## Expected behavior

> **DELETE THIS LINE BEFORE SUBMITTING** - Provide clear & concise description of what you expected to happen.

## Developer environment

> **DELETE THIS LINE BEFORE SUBMITTING** - Complete the following (*if ignored you'll be prompted for it before we can address your issue... save the time and provide it in your initial submission*).

- OS: [e.g. Windows 10, MacOS 10.15.x]
- Framework: [e.g. .NET Framework v3.x, .NET Core v3.x, Node.js v6/v8/v10/v12]
- Browser(s): [e.g. Chrome v79, Safari v22, FireFox v31, IE v11, Edge, Chromium Edge v79]
- Tooling: [e.g. VS Code, SPFx v1.10.0, Visual Studio 2019] 

### Additional context

> **DELETE THIS LINE BEFORE SUBMITTING** - Provide any other context about the problem.

*Thanks for your contribution! Sharing is caring.*

**Comment by bcameron1231** [comment]
There is not enough information provided to help you. 

Please fill out the Issue form template provided.

**Comment by thinkb4code** [comment]
Experiencing the same issue. Recently updated my packages to SPFx 1.10.0 version and using Node v10.16.2. Following were the steps I performed:

1. Scaffold the SPFx List View command set
2. Configured the solution for my environment
3. Changed one line of code i.e. onExecute method to `Dialog.alert(`${this.context.pageContext.web.permissions.hasPermission(SPPermission.manageWeb)}`);`

Now, when I gulp serve on SPFx List View Command Set solution, it bundle properly and start gulp server on localhost. But when I check the SharePoint list, it doesn't show command set button. When looking into developer console, it show following error:

![image](https://user-images.githubusercontent.com/24772450/79272240-04245a80-7e56-11ea-9cdc-d10402cccd54.png)


Not sure as of now if it is relevant or not. But on the site collection where I dont have this extension deployed, it doesnt output above error on console.
